README.md

```diff
@@ -7,20 +7,12 @@
 <p align="center">RPC-like client and server helpers for a magical end to end typed experience</p>
 
 <p align="center">
-  <a href="https://www.npmjs.com/package/@ts-rest/core">
-    <img src="https://img.shields.io/npm/v/@ts-rest/core.svg" alt="langue typescript"/>
-  </a>
+  <a href="https://www.npmjs.com/package/@ts-rest/core"><img src="https://img.shields.io/npm/v/@ts-rest/core.svg" alt="langue typescript"/></a>
```

Comment:
This was driving my OCD mad 😅

![Screenshot 2023-06-11 035334](https://github.com/ts-rest/ts-rest/assets/1728215/76eabeeb-8c38-45ee-bf10-1fb9845c40d4)